src/main/java/com/powsybl/openloadflow/dc/equations/DcEquationSystem.java

```diff
@@ -71,7 +71,7 @@ private static void createImpedantBranch(VariableSet variableSet, EquationSystem
                                              DcEquationSystemCreationParameters creationParameters, LfBranch branch,
                                              LfBus bus1, LfBus bus2) {
         if (bus1 != null && bus2 != null) {
-            boolean deriveA1 = creationParameters.isForcePhaseControlOffAndAddAngle1Var() && branch.getDiscretePhaseControl() != null;
+            boolean deriveA1 = creationParameters.isForcePhaseControlOffAndAddAngle1Var() && branch.canControlPhase();
```

Comment:
I prefer `hasPhaseControlCapability` for uniformity.

Reply:
Done